hack/test-update-storage-objects.sh

```diff
@@ -176,8 +175,7 @@ killApiServer
 
 KUBE_API_VERSIONS="${KUBE_NEW_API_VERSION},${KUBE_OLD_API_VERSION}"
 RUNTIME_CONFIG="api/all=false,api/${KUBE_OLD_API_VERSION}=true,api/${KUBE_NEW_API_VERSION}=true"
-# TODO: Switch to STORAGE_BACKEND_ETCD3 once we support it.
-startApiServer ${STORAGE_BACKEND_ETCD2} ${KUBE_NEW_STORAGE_VERSIONS} ${KUBE_STORAGE_MEDIA_TYPE_JSON}
+startApiServer ${STORAGE_BACKEND_ETCD3} ${KUBE_NEW_STORAGE_VERSIONS} ${KUBE_STORAGE_MEDIA_TYPE_JSON}
```

Comment:
Update the comment block above to say that this also tests rolling forward to etcd3?

Also, will this be flaky? Should we start and stop through etcd 2.3 first?